src/app/page-content/reviewing-page/add-review-form/add-review-form.component.ts

```diff
@@ -29,9 +29,11 @@ export class AddReviewFormComponent implements OnInit {
         .getAllStories()
         .filter(story => story.storyID == this.storyID)[0];
       const user = this.dataStore.getLoggedInUser();
-      user.addCredit(5 + Math.floor(Math.round(story.wordCount / 1000)));
+      // TODO: should there also be credit subtracted from the user who wrote the story being reviewed?
```

Comment:
I just realized this, looking through the credit logic, but I think we're missing the logic to take credit from the user who's story is being reviewed. We can probably add that later though.